File: Atv4_Repeticao/Livre/verifica.java
import java.util.Scanner;

public class verifica {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        int a, b;

        do {
            System.out.print("Digite o primeiro número (0 para sair): ");
            a = sc.nextInt();
            System.out.print("Digite o segundo número (0 para sair): ");
            b = sc.nextInt();

            if (a == 0 || b == 0) {
                System.out.println("Programa encerrado.");
                break;
            }

            if (a > b)
                System.out.println("O primeiro número é maior.");
            else if (a < b)
                System.out.println("O segundo número é maior.");
            else
                System.out.println("Os números são iguais.");
                break;
        } while (true);

        sc.close();
    }
}

File: Atv4_Repeticao/Livre/verifica.cs
using System;

class Program {
    static void Main() {
        int a, b;

        while (true) {
            Console.Write("Digite o primeiro número (0 para sair): ");
            a = int.Parse(Console.ReadLine());
            Console.Write("Digite o segundo número (0 para sair): ");
            b = int.Parse(Console.ReadLine());

            if (a == 0 || b == 0) {
                Console.WriteLine("Programa encerrado.");
                break;
            }

            if (a > b)
                Console.WriteLine("O primeiro número é maior.");
            else if (a < b)
                Console.WriteLine("O segundo número é maior.");
            else
                Console.WriteLine("Os números são iguais.");
                break;
        }
    }
}